widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/secureconversations/SecureConversationsWelcomeScreenTheme.kt

```diff
@@ -1,33 +1,30 @@
 package com.glia.widgets.view.unifiedui.theme.secureconversations
 
-import com.glia.widgets.view.unifiedui.theme.base.ButtonTheme
-import com.glia.widgets.view.unifiedui.theme.base.ColorTheme
-import com.glia.widgets.view.unifiedui.theme.base.HeaderTheme
-import com.glia.widgets.view.unifiedui.theme.base.LayerTheme
-import com.glia.widgets.view.unifiedui.theme.base.TextTheme
+import com.glia.widgets.view.unifiedui.theme.base.*
 import com.glia.widgets.view.unifiedui.theme.chat.AttachmentsPopupTheme
 import com.glia.widgets.view.unifiedui.theme.chat.FileUploadBarTheme
 
 internal data class SecureConversationsWelcomeScreenTheme(
-    val headerTheme: HeaderTheme?,
-    val welcomeTitleTheme: TextTheme?,
-    val titleImageTheme: ColorTheme?,
-    val welcomeSubtitleTheme: TextTheme?,
-    val checkMessagesButtonTheme: TextTheme?,
-    val messageTitleTheme: TextTheme?,
-    val messageTextViewNormalTheme: TextTheme?,
-    val messageTextViewDisabledTheme: TextTheme?,
-    val messageTextViewActiveTheme: TextTheme?,
-    val messageTextViewLayerTheme: LayerTheme?,
-    val enabledSendButtonTheme: ButtonTheme?,
-    val disabledSendButtonTheme: ButtonTheme?,
-    val loadingSendButtonTheme: ButtonTheme?,
-    val activityIndicatorColorTheme: ColorTheme?,
-    val messageWarningTheme: TextTheme?,
-    val messageWarningIconColorTheme: ColorTheme?,
-    val filePickerButtonTheme: ColorTheme?,
-    val filePickerButtonDisabledTheme: ColorTheme?,
-    val attachmentListTheme: FileUploadBarTheme?,
-    val pickMediaTheme: AttachmentsPopupTheme?,
-    val backgroundTheme: ColorTheme?
+    val headerTheme: HeaderTheme? = null,//
+    val welcomeTitleTheme: TextTheme? = null,//
+    val titleImageTheme: ColorTheme? = null,//
+    val welcomeSubtitleTheme: TextTheme? = null,//
+    val checkMessagesButtonTheme: TextTheme? = null,//
+    val messageTitleTheme: TextTheme? = null,//
+    val messageInputNormalTheme: TextInputTheme? = null,
+    val messageInputActiveTheme: TextInputTheme? = null,
+    val messageInputDisabledTheme: TextInputTheme? = null,
+    val messageInputErrorTheme: TextInputTheme? = null,
+    val messageInputHintTheme: TextTheme? = null,
+    val enabledSendButtonTheme: ButtonTheme? = null,
+    val disabledSendButtonTheme: ButtonTheme? = null,
+    val loadingSendButtonTheme: ButtonTheme? = null,
+    val activityIndicatorColorTheme: ColorTheme? = null,//
+    val messageWarningTheme: TextTheme? = null,//
+    val messageWarningIconColorTheme: ColorTheme? = null,//
+    val filePickerButtonTheme: ColorTheme? = null,//
+    val filePickerButtonDisabledTheme: ColorTheme? = null,//
+    val attachmentListTheme: FileUploadBarTheme? = null,
+    val pickMediaTheme: AttachmentsPopupTheme? = null,
+    val backgroundTheme: ColorTheme? = null//
```

Comment:
I think we can remove the `//` at the end of each line. Or add descriptions. 😉

Reply:
Thank you, those were leftovers.